src/main/java/br/com/sbk/sbking/networking/kryonet/KryonetClientFactory.java

```diff
@@ -4,7 +4,7 @@
 
 public class KryonetClientFactory {
   public static KryonetSBKingClient getRegisteredClient(SBKingClient sbkingClient) {
```

Comment:
As Sonarcloud pointed out, `sbkingClient` parameter is not necessary anymore

Reply:
Nice, thanks for noticing!